### ISSUE TITLE
Wrong time format in agenda

### ISSUE THREAD

**Issue by Core00077** [Zren/plasma-applet-eventcalendar#67]
The time display incorrectly on my agenda. Assuming a event from 8:00am to 9:00am.  It would only display the time `8-9` which is perfectly correct but not aligned. Expect it to be in the format of `8:00-9:00`.
![Screenshot_20190414_171000](https://user-images.githubusercontent.com/28624864/56090722-70cde780-5ed8-11e9-8040-9933a06d1002.png)
Here is a screenshot of it.

I am also wondering if it's my calendar's problem because I haven't see an issue the same as mine(Others' seemed just fine). This is my [calendar](https://calendar.google.com/calendar?cid=cWJoNHZiNDdsa25nNG9sb21pMjZxbzl1NDBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ). Hoping it would be of help.

really appreciate it for your time.

**Comment by Zren** [comment]
It's deliberate. [Event Flow Calendar](https://play.google.com/store/apps/details?id=com.syncedsynapse.eventflowwidget&hl=en_CA) used `5 - 6 PM` to simplify it when the time is "on the hour" as seen in this screenshot:

![image](https://user-images.githubusercontent.com/416367/56095505-ac9e8800-5eab-11e9-8ff8-8d310dbc8900.png)

I used `5 PM - 6 PM` just to make it simplier. From there I removed the AM/PM for 24h clocks when I added that feature. Now that I look at it, it seems the 24h clocks use `13:00 - 14:00` as seen in this screenshot:

![image](https://user-images.githubusercontent.com/416367/56095486-942e6d80-5eab-11e9-8d5c-6ea272c95b62.png)

Event Calendar doesn't have [Chinese translations](https://github.com/Zren/plasma-applet-eventcalendar/tree/master/package/translate), so we can get away with just editing the code for your case.

Open up `~/.local/share/plasma/plasmoids/org.kde.plasma.eventcalendar/contents/ui/LocaleFuncs.js` and change the line from:

```qml
timeFormat = i18nc("event time on the hour (24 hour clock)", "h")
```

to:

```qml
timeFormat = i18nc("event time on the hour (24 hour clock)", "h:mm")
```

Then to restart the panel, either relog or run: `killall plasmashell; kstart5 plasmashell`
